.grunt-config/webpack.js

```diff
@@ -111,6 +111,7 @@ const entry = {
 	'admin-top-bar': path.resolve( __dirname, '../modules/admin-top-bar/assets/js/admin.js' ),
 	'nested-elements': path.resolve( __dirname, '../modules/nested-elements/assets/js/editor/index.js' ),
 	'nested-tabs': path.resolve( __dirname, '../modules/nested-tabs/assets/js/editor/index.js' ),
+	'nested-tabs-html': path.resolve( __dirname, '../modules/nested-tabs-html/assets/js/editor/index.js' ),
```

Comment:
```suggestion
	'nested-tabs-v3': path.resolve( __dirname, '../modules/nested-tabs-html/assets/js/editor/index.js' ),
```
Maybe? v2 or v3. 'html' will be confusing in the future.